### DISCUSSION TITLE
I need some help regarding bandwidth

### DISCUSSION THREAD

**Discussion by mwdossantos** [vercel/vercel#4822] [Help]
Hi all,

I am currently on the Pro plan, and although the bandwidth indicates no limit, I saw on the docs that there is a 1TB limit (fair use policy). Currently, I have a couple of projects in the team account, and the bandwidth is at 229GB. Last time I checked it was at 187GB, which was three days ago.

<img width="437" alt="Screenshot 2020-07-09 at 12 09 45" src="https://user-images.githubusercontent.com/8517723/87028244-a1392a00-c1de-11ea-8ab5-63301181c88a.png">

I am a bit worried that I am going to go hit the 1TB limit unintentionally.

What is the best way to find out what is eating up all that bandwidth? Or should I not worry at all?

I eagerly await your answer.

Best regards,
Matthew

**Comment by samcx** [comment]
Hello,

This is, unfortunately, not available in the UI, but I can take a look with internal tools. What's your Vercel team slug?

**Reply by haywirez** [reply]
Would it be possible to expose the usages via the API? I'm also wondering whether the fair-use limits are per-deployment/domain, or an account total?
